lib/bolt/config.rb

```diff
@@ -286,5 +286,27 @@ def validate
         impl.validate(@transports[transport])
       end
     end
+
+    # Check if there is a case-insensitive match to the path
+    def check_path_case(type, paths)
+      return if paths.nil?
+      matches = matching_paths(paths)
+
+      if matches.any?
+        msg = "WARNING: Bolt is case sensitive when specifying a #{type}. Did you mean:\n"
```

Comment:
This spaces things a little weirdly. I think the colon is enough separation without the newline and the tab before the path. So just "WARNING: Bolt is case sensitive when specifying a #{type}. Did you mean: #{path}"